packages/core/src/test/awsService/cloudWatchLogs/registry/liveTailSession.test.ts

```diff
@@ -65,6 +83,36 @@ describe('LiveTailSession', async function () {
         )
     })
 
+    it('startLiveTail sends command to CWL client with AbortSignal', async function () {
+        const sendCommandMock = sandbox.stub(CloudWatchLogsClient.prototype, 'send').callsFake(function () {
+            return buildLiveTailCommandOutput(testLogGroupArn)
+        })
+        const session = buildLiveTailSession({ type: 'all' }, undefined)
+        const stream = await session.startLiveTailSession()
+
+        assert.strictEqual(sendCommandMock.calledOnce, true)
+        assert.strictEqual(
+            sendCommandMock.calledWith(
```

Comment:
instead of mocking many things just to test that parameters are sent, can you stub fake responses and let the code actually work on the response. that tests the full codepath, instead of just tiny parts of it (and requiring lots of test code)

Reply:
Sorry, I don't think I understand the requested change to be made here. 
The function in test here is setting some metadata on the LiveTailSession, calling StartLiveTail API, and returning the stream. I'm only mocking one thing and that is stubbing a fake response from CWL API, and testing it's being returned.
The AbortSignal object is not exposed to upstream callers, so validating that it is passed into the sendCommand is only doable via testing this method directly. Given that firing the abortSignal is critical to closing a LiveTail session I feel like having this validation is worth having the test code.